src/app/configuration/configuration.service.ts

```diff
@@ -0,0 +1,200 @@
+import { FormBuilder, FormGroup, FormControl, Validators } from '@angular/forms';
+import { CouchService } from '../shared/couchdb.service';
+import { ValidatorService } from '../validators/validator.service';
+import { PlanetMessageService } from '../shared/planet-message.service';
+import { Router, ActivatedRoute } from '@angular/router';
+import { UserService } from '../shared/user.service';
+import { switchMap, mergeMap } from 'rxjs/operators';
+import { Observable, forkJoin } from 'rxjs';
+import { findDocuments } from '../shared/mangoQueries';
+import { debug } from '../debug-operator';
+
+
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ConfigurationService {
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private couchService: CouchService,
+    private planetMessageService: PlanetMessageService,
+    private validatorService: ValidatorService,
+    private router: Router,
+    private route: ActivatedRoute,
+    private userService: UserService) { }
+
+
+  getFeedbackSyncUp(environment, configuration, credentials, adminName) {
+    return {
+      '_id': 'feedback_to_parent',
+      'source': {
+        'headers': {
+          'Authorization': 'Basic ' + btoa(credentials.name + ':' + credentials.password)
+        },
+        'url': environment.couchAddress + 'feedback'
+      },
+      'target': {
+        'headers': {
+          'Authorization': 'Basic ' + btoa(adminName + ':' + credentials.password)
+        },
+        'url': 'https://' + configuration.parentDomain + '/feedback'
+      },
+      'create_target': false,
+      'continuous': true,
+      'owner': credentials.name
+    };
+  }
+
+  getFeedbackSyncDown(feedbackSyncUp, configuration) {
```

Comment:
we will have to do some rework as we do not want feedback sync in configuration now. But that changes has not been merged to master yet.